src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: ../../layouts/ContentLayout.astro
+---
+
+# Ce qui caractérise le top 20% des devs
+
+## Introduction
+
+Lorsque l'on parle des développeurs, il est fréquent de vouloir comprendre ce qui distingue les meilleurs, le fameux top 20%, du reste des développeurs.
+
+Que font-ils différemment ?
+
+Quels sont leurs traits de caractère, leurs habitudes ou leurs compétences spécifiques qui les propulsent au sommet de leur domaine ?
+
+Je suis Oumar, j’ai rejoins le monde de la programmation en septembre 2022 et c’est là tout un tas de question que je me suis posé (et que je me pose toujours d’ailleurs pour sainement progresser 😉).
+
+Je vous propose donc dans ce blog post de vous partager les conseils reçus, de mes paires plus “seniors”. Cela me sers beaucoup au quotidien et je l’espère vous servira également. Allez on se lance !
+
+## Oui, la technique ça compte
+
+### La curiosité et la résilience
+
+Aussi technique soi le sujet cela ne les refrènent pas le moins du monde, c’est un nouvel obstacle parmi tant d’autres à surmonter. Cela est même souvent source de plaisir pour beaucoup.
+
+Notre métier est sacrément dur faut le dire et le reconnaître !
+
+En effet, les meilleurs développeurs sont souvent des personnes curieuses-résilientes (j’insiste sur la mise en commun de ces deux caractéristiques) qui s'intéressent à de nouvelles technologies, outils ou langages de programmation. Ils aiment expérimenter avec de nouvelles idées et ne se contentent pas de suivre les méthodes établies. Ils cherchent toujours à comprendre comment les choses fonctionnent en profondeur. C’est souvent de la que ressort l’un des deux groupes du top 20%, que j’entends souvent qualifier “d’innovateurs”. On parlera du second groupe, dans la prochaine caractéristique.
+
+### Approche orientée problème à la solution (et non l’inverse)
+
+Ils ont une approche orientée problème, ce qui signifie qu'ils se concentrent sur la résolution de problèmes spécifiques plutôt que sur l'application d'une technologie en elle-même.
```

Comment:
You should give an example (if possible)

Reply:
Done. 🙂